hips/tiles/tile.py

```diff
@@ -95,7 +95,7 @@ def read(cls, meta: HipsTileMeta, filename: str = None) -> 'HipsTile':
 
         if meta.file_format == 'fits':
             hdu_list = fits.open(str(path))
-            data = hdu_list[0].data
+            data = hdu_list[0].data.astype('float')
```

Comment:
DSS2Red FITS tiles are encoded as 16-bits integers (BITPIX = 16), so that makes sense to me.